ddl/delete_range.go

```diff
@@ -238,7 +238,24 @@ func insertJobIntoDeleteRangeTable(ctx sessionctx.Context, job *model.Job) error
 		}
 	case model.ActionDropTable, model.ActionTruncateTable:
 		tableID := job.TableID
-		startKey := tablecodec.EncodeTablePrefix(tableID)
+		// The startKey here is for compatibility with previous versions.
+		var startKey kv.Key
```

Comment:
Why not treat the `endKey` in the same way?

Reply:
@shenli Because the old version did not handle ` endKey `.

Reply:
You probably need to put "Because the old version did not handle endKey." to comment too.